models/handpose_estimation_mediapipe/demo.py

```diff
@@ -0,0 +1,162 @@
+
+    # Print results
+    if print_result:
+        print('conf: {:.2f}'.format(conf))
+        print('hand box: {}'.format(bbox))
+        print('hand landmarks: ')
+        for l in landmarks:
+            print('\t{}'.format(l))
+        print('----------------------')
+
+    # Draw line between each key points
+    cv.line(output, landmarks[0], landmarks[1], (255, 255, 255), 2)
+    cv.line(output, landmarks[1], landmarks[2], (255, 255, 255), 2)
+    cv.line(output, landmarks[2], landmarks[3], (255, 255, 255), 2)
+    cv.line(output, landmarks[3], landmarks[4], (255, 255, 255), 2)
+
+    cv.line(output, landmarks[0], landmarks[5], (255, 255, 255), 2)
+    cv.line(output, landmarks[5], landmarks[6], (255, 255, 255), 2)
+    cv.line(output, landmarks[6], landmarks[7], (255, 255, 255), 2)
+    cv.line(output, landmarks[7], landmarks[8], (255, 255, 255), 2)
+
+    cv.line(output, landmarks[0], landmarks[9], (255, 255, 255), 2)
+    cv.line(output, landmarks[9], landmarks[10], (255, 255, 255), 2)
+    cv.line(output, landmarks[10], landmarks[11], (255, 255, 255), 2)
+    cv.line(output, landmarks[11], landmarks[12], (255, 255, 255), 2)
+
+    cv.line(output, landmarks[0], landmarks[13], (255, 255, 255), 2)
+    cv.line(output, landmarks[13], landmarks[14], (255, 255, 255), 2)
+    cv.line(output, landmarks[14], landmarks[15], (255, 255, 255), 2)
+    cv.line(output, landmarks[15], landmarks[16], (255, 255, 255), 2)
+
+    cv.line(output, landmarks[0], landmarks[17], (255, 255, 255), 2)
+    cv.line(output, landmarks[17], landmarks[18], (255, 255, 255), 2)
+    cv.line(output, landmarks[18], landmarks[19], (255, 255, 255), 2)
+    cv.line(output, landmarks[19], landmarks[20], (255, 255, 255), 2)
+
+    for p in landmarks:
+        cv.circle(output, p, 2, (0, 0, 255), 2)
+
+    return output
+
+
+if __name__ == '__main__':
+    # palm detector
+    palm_detector = MPPalmDet(modelPath='../palm_detection_mediapipe/palm_detection_mediapipe_2022may.onnx',
+                              nmsThreshold=0.3,
+                              scoreThreshold=0.8,
+                              backendId=args.backend,
+                              targetId=args.target)
+    # handpose detector
+    handpose_detector = MPHandPose(modelPath=args.model,
+                                   confThreshold=args.conf_threshold,
+                                   backendId=args.backend,
+                                   targetId=args.target)
+
+    # If input is an image
+    if args.input is not None:
+        image = cv.imread(args.input)
+
+        # Palm detector inference
+        palms = palm_detector.infer(image)
+        # Estimate the pose of each hand
+        for _, palm in enumerate(palms):
+            # Handpose detector inference
+            handpose = handpose_detector.infer(image, palm)
+            if handpose is None:
+                continue
+            # Draw results on the input image
+            image = visualize(image, handpose, True)
```

Comment:
Results should be collected in an numpy array. Call `visualize` once to draw all possible hand poses.